lib/compiler.js

```diff
@@ -1,5 +1,5 @@
 var assert = require('assert');
-var cloneDeep = require('lodash.clonedeep');
+var cloneDeep = require('lodash/lang/cloneDeep');
```

Comment:
FWIW, compiler is not bundled for the browser, so the following is acceptable too: 
```js
var cloneDeep = require('lodash').cloneDeep;
```